projects/go-tester/src/app/components/off-canvas-test/off-canvas-test.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="go-container">
-  <go-off-canvas-header-actions>
+  <!-- <go-off-canvas-header-actions>
```

Comment:
this throws up error when running 'tester'. So commented it for now.

Reply:
We probably shouldn't be committing commented out code, in most cases. What was the error for this? I just pulled `dev` and ran `npm run tester` and it completed successfully for me.

Reply:
@grahamhency : Apologies, i should have posted a screenshot.
When you run `npm run tester` and check the Chrome console, this is the error you would notice:
![image](https://user-images.githubusercontent.com/3825309/95310999-af184d00-08aa-11eb-83bc-8307ff26e08f.png)
I could not find any such component in the project, shall I remove the commented code entirely?

Reply:
@Ashygoyal yeah that's fine then. I didn't see it either. Weird.

Reply:
@grahamhency : removed the commented code in latest commit. I think that was the only pending part, you can still review and let me know.